vortex-api/src/main/java/mil/army/usace/hec/vortex/io/VariableDsReader.java

```diff
@@ -205,6 +206,9 @@ public VortexGrid read(int index) {
     }
 
     private VortexGrid createDTO(float[] data, ZonedDateTime startTime, ZonedDateTime endTime, Duration interval) {
+        VortexVariable vortexVariable = VortexVariable.fromName(variableDS.getShortName());
+        // FIXME: How about VortexDataType
+
         return VortexGrid.builder()
```

Comment:
For reading in NetCDF data -- should we parse the `VortexDataType` via `CELL_METHODS`, and if it doesn't exist, then infer it from the `VortexVariable`?

Reply:
Reading from `CELL_METHODS` seems safe. Inferring can be done but is risky to the point I would consider treating as `UNKNOWN`. Maybe you can jog my memory on how we are using the data type.